charts/nr-k8s-otel-collector/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for nr-k8s-otel-collector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+kube-state-metrics:
```

Comment:
I would suggest to remove the label enabling the autodiscovery from prometheus. We have talked about it countless of times for nri-bundle and we were able to do so to avoid breaking changes.

Reply:
https://github.com/newrelic/helm-charts/pull/376

Reply:
Not sure what this means, where do we have a label enabling autodiscovery from prometheus? this setting only enabled installing kube-state-metrics, nothing to do w/ autodiscovery

Reply:
There is a toggle in the KSM chart `prometheusScrape: true`
That allows the KSM service to be "autodiscovered" by the label "prometheus.io/scrape". This is not bad per se, but if the customer have installed a Prometheus server or POMI or a prometheus agent they might end up scraping a log of additional metrics without realising

Reply:
https://github.com/prometheus-community/helm-charts/blob/f29a2580f698ab6cac895785fe72886c8a42fec7/charts/kube-state-metrics/values.yaml#L2

Reply:
Fixed :)